Assets/MixedRealityToolkit.Providers/WindowsMixedReality/Shared/Definitions/IWindowsMixedRealityUtilitiesProvider.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Microsoft.MixedReality.Toolkit.WindowsMixedReality
+{
+    public interface IWindowsMixedRealityUtilitiesProvider
```

Comment:
Nit: Might also be worth adding some doccomments to this to (if only to help future generations understand what should go on this interface, and what shouldn't)

It's pretty clear from the contents what the things inside of it are now, though these are the types of interfaces that can balloon to much larger things going forward without guidance about initial intent (i.e. maybe ballooning is also fine, etc)

Reply:
This one shouldn't increase very much, if at all, and really only needs to exist while we support both legacy Unity XR and XR SDK.

I'll add some comments though!

Reply:
Added a summary with some remarks

Reply:
> really only needs to exist while we support both legacy Unity XR and XR SDK

That said, this is a good pattern to maintain going forward as the engine may change in ways that the provider needs to accomodate.